### ISSUE TITLE
[QUESTION] Web Support Roadmap

### ISSUE THREAD

**Issue by CorinneKelly** [react-navigation/react-navigation#9223]
I see in your [5.x documentation](https://reactnavigation.org/docs/web-support) that Web Support is still in the experimental stage and in your [v6 roadmap](https://github.com/react-navigation/react-navigation/issues/8981) there is no mention of web support. Do you have a reasonable estimation on when web support will be stable? Thanks!

**Comment by lanman2000** [comment]
Web support works. What's missing is the ability to use the history API to attach back button.

**Comment by sergeymild** [comment]
Yes , I have a problem with history api, i am using a react-navigation in web and it's works perfect, but if i refresh a page i can't go back with navigation.goBack(). I seeing this error `The action 'GO_BACK' was not handled by any navigator.`. Maybe there is some workarounds how solve this problem?

**Comment by jakedeg** [comment]
We're having the same problem. We're excited to use web support in production, but this feature gap is tricky. Any ideas for workarounds would be appreciated.
